gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/tasks/snapshots/LocalSnapshots.kt

```diff
@@ -147,6 +149,10 @@ abstract class LocalSnapshots : DefaultTask() {
       val output = shell(instrumentationArgs)
       logger.lifecycle(output)
 
+      logger.lifecycle("Dumping logcat logs to file")
```

Comment:
This intentional to leave in? Seems maybe this should be a debug setting controlled by an arg.